prowler/providers/aws/services/storagegateway/storagegateway_service.py

```diff
@@ -0,0 +1,106 @@
+
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def __describe_nfs_file_shares__(self, regional_client):
+        logger.info("StorageGateway - Describe NFS FileShares...")
+        try:
+            for fileshare in self.fileshares:
+                if fileshare.file_share_type == "NFS":
+                    filesharelist = []
+                    filesharelist.append(fileshare.file_share_arn)
+                    response = regional_client.describe_nfs_file_shares(
+                        FileShareARNList=filesharelist
+                    )
+                    if "Tags" in response["NFSFileShareInfoList"][0]:
+                        fileshare.tags = response["NFSFileShareInfoList"][0]["Tags"]
+                    else:
+                        fileshare.tags = []
+                    fileshare.kms = response["NFSFileShareInfoList"][0]["KMSEncrypted"]
+                    if response["NFSFileShareInfoList"][0]["KMSEncrypted"]:
+                        fileshare.kms_key = response["NFSFileShareInfoList"][0][
+                            "KMSKey"
+                        ]
+                    else:
+                        fileshare.kms_key = ""
+
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def __describe_smb_file_shares__(self, regional_client):
+        logger.info("StorageGateway - Describe SMB FileShares...")
+        try:
+            for fileshare in self.fileshares:
+                if fileshare.file_share_type == "SMB":
+                    filesharelist = []
+                    filesharelist.append(fileshare.file_share_arn)
+                    response = regional_client.describe_smb_file_shares(
+                        FileShareARNList=filesharelist
+                    )
+                    if "Tags" in response["SMBFileShareInfoList"][0]:
+                        fileshare.tags = response["SMBFileShareInfoList"][0]["Tags"]
+                    else:
+                        fileshare.tags = []
+                    fileshare.kms = response["SMBFileShareInfoList"][0]["KMSEncrypted"]
+                    if response["SMBFileShareInfoList"][0]["KMSEncrypted"]:
+                        fileshare.kms_key = response["SMBFileShareInfoList"][0][
+                            "KMSKey"
+                        ]
+                    else:
+                        fileshare.kms_key = ""
+
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+
+class FileShare(BaseModel):
+    id: str
+    file_share_arn: str
+    gateway_arn: str
+    region: str
+    file_share_type: str
```

Comment:
```suggestion
    _type: str
```

Reply:
_type caused some errors so changed it to fs_type and that worked. Hopefully that solves comment.